source/installer.py

```diff
@@ -582,6 +590,7 @@ def tryRemoveFile(path,numRetries=6,retryInterval=0.5,rebootOK=False):
 			return
 		except OSError:
 			pass
+		log.debugWarning(f"Failed to delete file {tempPath}, attempt {count}/{numRetries}")

```

Comment:
Is there a reason why this is not in the `except` blog? If it would be moved there we could also log the exception which caused the deletion to fail.